tests/api/utils/test_dict_utils.py

```diff
@@ -0,0 +1,101 @@
+from snowflake.cli.api.utils.dict_utils import deep_merge_dicts, deep_traverse
+
+
+def test_merge_dicts_empty():
+    test_dict = {}
+    deep_merge_dicts(test_dict, {})
+    assert test_dict == {}
+
+
+def test_merge_dicts_recursive_map():
+    test_dict = {"a": "a1", "b": "b1", "c": {"d": "d1", "e": "e1"}}
```

Comment:
nit: woah that's hard to read. Guessing the formatted messed with what would otherwise be a very clear test. How about cheating then and parsing the dict from a multi-line JSON structure? It's really hard to do the merge in my head based on the single-line structures here.

Reply:
yea - it's very annoying. Let me see if there is a way to skip the formatting on this, otherwise, I'll use Json as you suggested.

Reply:
Can be avoided by using # fmt: off / #fmt: on